lib/cli/scripts/plugins/aae-plugin.ts

```diff
@@ -0,0 +1,69 @@
+import { PluginInterface } from './plugin-model';
+import { logger } from '../logger';
+import { PluginConfiguration } from './plugin-config';
+
+export class ProcessAutomationPlugin {
+    config: PluginConfiguration;
+
+    constructor(
+        private plugInInfo: PluginInterface,
+        private alfrescoJsApi: any
+    ) {
+        this.config = new PluginConfiguration(
+            this.plugInInfo,
+            this.alfrescoJsApi,
+            true
```

Comment:
a method should never have a boolean. 
if you have a boolean means that you have 2 behaviour... so you can create 2 classes.

Reply:
Updated